db/data_migration/20200205160940_remove_slugs_for_file_and_external_attachments.rb

```diff
@@ -0,0 +1,4 @@
+Attachment
+  .where.not(type: "HtmlAttachment")
+  .where.not(slug: nil)
```

Comment:
Minor - this can be where.not(type: "HtmlAttachment", slug: nil)

Reply:
Nice, I normally like to put my where clauses on separate lines to be super explicit as I think it reads better but thats just personal preference!